src/diffusers/pipelines/stable_diffusion_xl/pipeline_stable_diffusion_xl_img2img.py

```diff
@@ -781,12 +808,14 @@ def __call__(
 
         # 5. Prepare timesteps
         self.scheduler.set_timesteps(num_inference_steps, device=device)
-        timesteps, num_inference_steps = self.get_timesteps(num_inference_steps, strength, device)
+        timesteps, num_inference_steps = self.get_timesteps(num_inference_steps, first_inference_step, strength, device)
```

Comment:
```suggestion
        if denoising_start > 0.0:
            logger.info(f"`strength`: {strength} is overwritten with `denoising_start`: {denoising_start}.")
            strength = denoising_start
            
        timesteps, num_inference_steps = self.get_timesteps(num_inference_steps, strength, device)
```

Reply:
actually, inside get_timesteps, the strength value is ignored.